controller/qos_manager.py

```diff
@@ -158,7 +160,7 @@ def get_rules(self):
         which triggers every function subscribed to the ofp_event.EventOFPFlowStatsReply
         event.
         """
-        r = requests.get("%s/qos/rules/%016x" % (QoSManager.CONTROLLER_BASEURL, self.__datapath.id))
+        r = requests.get("%s/qos/rules/all" % QoSManager.CONTROLLER_BASEURL)
```

Comment:
This one might be worth a second thought...
Should use the same solution as in `set_rules()`.